### DISCUSSION TITLE
为啥在线生成windows的应用，生成后运行没窗口呀，试了很多遍了。。。。

### DISCUSSION THREAD

**Discussion by Eufemlola** [tw93/Pake#850] [Q&A]
为啥在线生成windows的应用，生成后运行没窗口呀，试了很多遍了。。。。

**Comment by puzzle9** [comment]
你把日志丢出来

**Reply by Eufemlola** [reply]
稍等

**Reply by Eufemlola** [reply]
大哥，帮忙看看嘛

**Comment by Eufemlola** [comment]
![Uploading {9E404F3C-7CD3-4C48-BEB3-A56841F44777}.png…]()

**Comment by Eufemlola** [comment]
![{9E404F3C-7CD3-4C48-BEB3-A56841F44777}](https://github.com/user-attachments/assets/fc22e0aa-0143-4b0e-80d5-e902dd7811e2)
1

**Comment by Eufemlola** [comment]
本地执行，卡在这里
![{DDA8FBEA-A703-4405-B454-75A07368DCC7}](https://github.com/user-attachments/assets/7c7e8886-f2fe-43b0-bf0b-49ccfdc2f104)

**Comment by Eufemlola** [comment]
![{9E404F3C-7CD3-4C48-BEB3-A56841F44777}](https://github.com/user-attachments/assets/1d7fd781-84f2-456b-9329-d4d32d45849e)
安装后也是一样 没显示窗口 打不开

**Comment by Eufemlola** [comment]
![Uploading {7673515A-D7B9-4C71-84C0-441B97F83C99}.png…]()

**Comment by Eufemlola** [comment]
1
![{7673515A-D7B9-4C71-84C0-441B97F83C99}](https://github.com/user-attachments/assets/68289ac1-ac03-430e-b7b5-11225275f27d)

**Reply by puzzle9** [reply]
看起来你是win10 本地打包的
后面加 `--debug` 打包后运行试下
看有什么运行日志不

**Comment by rongrong1000** [comment]
手动安装一下 webview2 (edge) 组件试试
